docs/general/pull-request.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
Lets put this MD file inside  docs/github folder instead of docs/general

Reply:
Changed